core/Node.js

```diff
@@ -661,6 +661,32 @@ Node.prototype.removeComponent = function removeComponent (component) {
 };
 
 /**
+ * Removes a node's subscription to a particular UIEvent. All components 
+ * on the node will have the opportunity to remove all listeners depending
+ * on this event.
+ *
+ * @method
+ *
+ * @param {String} eventName the name of the event
+ *
+ * @return {undefined} undefined
+ */
+Node.prototype.removeUIEvent = function removeUIEvent (eventName) {
+    var UIEvents = this.getUIEvents();
+    var components = this._components;
+    var component;
+
+    var index = UIEvents.indexOf(eventName);
```

Comment:
Bikeshedding: Can we change this to `this.value.UIEvents`? Since we're mutating the array I think it would be better to make it obvious that we're mutating the actual array, not a clone returned by the getter. It's slightly shorter and I don't think it's less readable.